mobile/src/reducers/people/people.reducer.ts

```diff
@@ -0,0 +1,48 @@
+import { Reducer } from 'redux';
+import { Map, Set } from 'immutable';
+import { combineEpics } from 'redux-observable';
+import { NavigationAction } from 'react-navigation';
+
+import { User } from '../user/user.model';
+import { Employee } from '../organization/employee.model';
+import { OrganizationActions, loadEmployeesForDepartment } from '../organization/organization.action';
+import { UserActions } from '../user/user.action';
+import { PeopleActions } from './people.action';
+import { EmployeeMap } from '../organization/employees.reducer';
+
+export interface PeopleState {
+    employees: EmployeeMap;
+    employeesDepartmentSubsetFilterCallback: any;
+    employeesRoomSubsetFilterCallback: any;
+    employeesSubsetFilterCallback: any;
+}
+
+const initState: PeopleState = {
+    employees: Map(),
+    employeesDepartmentSubsetFilterCallback: function(employee: Employee) {
+        return employee.employeeId === '';
+    },
+    employeesRoomSubsetFilterCallback: function(employee: Employee) {
+        return employee.employeeId === '';
+    },
+    employeesSubsetFilterCallback: function(employee: Employee) {
+        return employee.employeeId === '';
+    }
+};
+
+export const peopleReducer: Reducer<PeopleState> = (state = initState, action: PeopleActions) => {
+    switch (action.type) {
+        case 'NAVIGATE-PEOPLE-DEPARTMENT':
+            return {...state, employeesDepartmentSubsetFilterCallback: function(employee: Employee) {
```

Comment:
Instead of using anonymous function here, please move this functions above in the file. The reason is that each time you get here it returns a new instance of the function, which will cause a rerender in components (if, you, say, call this event multiple times being on the same screen... which can occur)